demo/markdownlint-browser.js

```diff
@@ -4168,6 +4168,8 @@ module.exports = {
     "function": function MD040(params, onError) {
         let allowed = params.config.allowed_languages;
         allowed = Array.isArray(allowed) ? allowed : [];
+        let extra = params.config.extra;
+        extra = (extra === undefined) ? true : !!extra;
```

Comment:
I prefer to default unset parameters to false. Could this be reframed as "language_only"? (Not sure I love that name, but you get the idea.)